### PR TITLE
Feature/add php composer workflows

### PR DESCRIPTION
**Pull request by robmellett**
## Description of changes (updated or new workflows)

Added some workflows for using Composer, which is the main dependency manager within the PHP community. 

I'm not sure who to attribute the author fields to - so give me a shout if I've labeled them incorrect.

image

### CODE REVIEW

File: specs/composer/show_all_packages.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Show a list of available packages
```
Comment (alokedesai) [inline specs/composer/show_all_packages.yml:2]: Mind making the name here and below clearer that this is related to composer? So `Show a list of available composer packages` instead?

**Review by alokedesai — COMMENTED** [review]
Looks great! Just a few small comments on how we can make this clearer that it's related to `composer`. Thank you for contributing 👍🏽

**Comment by alokedesai** [comment]
Hey @robmellett! Friendly ping on this--would love to get these merged or close out if they are no longer relevant

**Comment by robmellett** [comment]
> @alokedesai and @robmellett I'd be happy to take over this PR if y'all like, as I just submitted another PHP for some PHP workflows.
> 
> I'll check back in a week if I haven't heard anything. 🙂

Yeap go ahead. I actually forgot about this. I'll close this PR if you like.

**Comment by n8finch** [comment]
> Yeap go ahead. I actually forgot about this. I'll close this PR if you like.

Cheers, Rob, I've added your files to #142 (keeping your attribution as well).

**Pull request closed without merging**